mmv1/third_party/terraform/tests/resource_database_migration_service_connection_profile_test.go

```diff
@@ -0,0 +1,78 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatabaseMigrationServiceConnectionProfile_update(t *testing.T) {
+	t.Parallel()
+
+	suffix := map[string]interface{}{
+		"random_suffix": RandString(t, 10),
+	}
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatabaseMigrationServiceConnectionProfile_basic(suffix),
+			},
+			{
+				ResourceName:            "google_database_migration_service_connection_profile.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"connection_profile_id", "location", "mysql.0.password"},
+			},
+			{
+				Config: testAccDatabaseMigrationServiceConnectionProfile_update(suffix),
+			},
+			{
+				ResourceName:            "google_database_migration_service_connection_profile.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"connection_profile_id", "location", "mysql.0.password"},
+			},
+		},
+	})
+}
+
+func testAccDatabaseMigrationServiceConnectionProfile_basic(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_database_migration_service_connection_profile" "default" {
+	location = "us-central1"		
+	connection_profile_id = "tf-test-dbms-connection-profile%{random_suffix}"
+	display_name          = "tf-test-dbms-connection-profile-display%{random_suffix}"
+	labels	= { 
+					testanno = "testanno" 
+				}
+	mysql {
+	  host = "10.20.30.40"
+	  port = 3306
+	  username = "tf-test-dbms-test-user%{random_suffix}"
+	  password = "tf-test-dbms-test-pass%{random_suffix}"
+	}
+}
+`, context)
+}
+
+func testAccDatabaseMigrationServiceConnectionProfile_update(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_database_migration_service_connection_profile" "default" {
+	location = "us-central1"		
+	connection_profile_id = "tf-test-dbms-connection-profile%{random_suffix}"
+	display_name          = "tf-test-dbms-connection-profile-display%{random_suffix}"
```

Comment:
`display_name` and `labels` seem to be top-level updatable fields. we should test updating them here.